test/e2e/specs/customer_admin.go

```diff
@@ -104,5 +107,114 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 		Expect(kerrors.IsForbidden(err)).To(Equal(true))
 	})
 
+	It("should be able to manage Quotas, LimitRanges, and EgressNetworkPolicies", func() {
+		// create a project as an end user
+		// add quota as customer-admin
+		// verify it was added
+		namespace, err := randomstring.RandomString("abcdefghijklmnopqrstuvwxyz0123456789", 5)
+		Expect(err).ToNot(HaveOccurred())
+		namespace = "e2e-test-" + namespace
+		err = cli.Client.EndUser.CreateProject(namespace)
+		Expect(err).ToNot(HaveOccurred())
+		defer cli.Client.EndUser.CleanupProject(namespace)
+
+		resQuota := v1.ResourceQuota{
+			Spec: v1.ResourceQuotaSpec{
+				Hard: v1.ResourceList{
+					"configmaps":             resource.MustParse("1"),
+					"persistentvolumeclaims": resource.MustParse("1"),
+				},
+			},
+		}
+		resQuota.ObjectMeta.Name = "testquota"
```

Comment:
nit: you name here `testquota` but below you use abbreviations for limits `testlr`.

Reply:
Made the test names align.